LCAPI.Core/Patches/MenuManager.cs

```diff
@@ -16,7 +16,7 @@ internal class MenuManagerPatches
     [HarmonyPatch("Awake")]
     [HarmonyPostfix]
     [HarmonyWrapSafe]
-    public static void Awake_Postfix(MenuManager __instance)
+    private static void Awake_Postfix(MenuManager __instance)
```

Comment:
To prevent other assembly to access it by normal meaning.
This won't break anything because Harmony will found your method regardless of the visibility setting of it. That's also the reason why you can patch internal/private method

Reply:
I just had a look in the TerminalCommands API. We're also using public there. Please change this to public so we use a common standard.